sdv/sampling/independent_sampler.py

```diff
@@ -0,0 +1,166 @@
+"""Independent Samplers."""
+import logging
+from copy import deepcopy
+
+import pandas as pd
+
+LOGGER = logging.getLogger(__name__)
+
+
+class BaseIndependentSampler():
+    """Independent sampler mixin.
+
+    Args:
+        metadata (sdv.metadata.multi_table.MultiTableMetadata):
+            Multi-table metadata representing the data tables that this sampler will be used for.
+        table_synthesizers (dict):
+            Dictionary mapping each table to a synthesizer. Should be instantiated and passed to
+            sampler by the multi-table synthesizer using this sampler.
+        table_sizes (dict):
+            Dictionary mapping each table to its size. Should be instantiated and passed to the
+            sampler by the multi-table synthesizer using this sampler.
+    """
+
+    def __init__(self, metadata, table_synthesizers={}, table_sizes={}):
```

Comment:
Overall is better to use `None` as `default` when optional, then do a simple `self._table_synthesizers = table_synthesizers or {}`.
However, I wonder if this are optional here 🤔